### PR TITLE
Data transfer documentation for RTD 2.0

### PR DESCRIPTION
**Pull request by joseph-cooke**
https://github.com/northeastern-rc/rc-public-documentation/pull/42

### CODE REVIEW

**Comment by visionjo** [comment]
I converted several of the sections to tabs (two sets, one for terminal, and the other for GUI). Let me know what you think @joseph-cooke

File: docs/source/using-discovery/transferringdata.md [hunk 1]
```diff
@@ -47,14 +127,23 @@ The Files menu in Open onDemand (OOD) also allows you to transfer files. See [Us
...
-You will now be connected to the transfer node and can transfer files through MobaXterm.
+You will now be connected to the transfer node and can transfer files through MobaXterm. Please refer to [MobaXterm] for further information.
 
-**To transfer data using FileZilla:**
+
+::::
+::::{tab-item} FileZilla
+You can use Firefox to transfer data to and from the HPC.
 
 1. Open FileZilla.
```
Comment (visionjo) [inline docs/source/using-discovery/transferringdata.md:137]: Should we provide a link for download?

File: docs/source/using-discovery/transferringdata.md [hunk 2]
```diff
@@ -1,44 +1,124 @@
...
 
-**To transfer data using MobaXterm:**
+:::::{tab-set}
+::::{tab-item} OOD's File Explorer
+
+You can use OOD's File Explorer application to transfer data from different directories on the HPC and also to transfer data to and from your local machine to the HPC. For more information to complete this please see {ref}`file-explorer`.
+::::
+::::{tab-item} MobaXterm
+
+You can use MobaXterm to transfer data to and from the HPC.
```
Comment (visionjo) [inline docs/source/using-discovery/transferringdata.md:121]: Link for download?

**Review by visionjo — CHANGES_REQUESTED** [review]
Couple of comments! Looks great!

Be sure to check to see if the tabs work! Modify as you see fit.

File: docs/source/using-discovery/transferringdata.md (hunk 2 above)
Comment (joseph-cooke) [reply docs/source/using-discovery/transferringdata.md]: I added the link to MobaXterm as well. Thank you!!

**Comment by joseph-cooke** [comment]
Thank you for adding in the tabs @visionjo, it works really well and breaks the page up nicely. Thank you!!

**Review by visionjo — APPROVED** [review]
Nice!

**Pull request merged**